backend/audit/fixtures/__init__.py

```diff
@@ -1,9 +1,9 @@
-from .single_audit_checklist import (
-    load_single_audit_checklists,
-    load_single_audit_checklists_for_email_address,
-)
+# from .single_audit_checklist import (
```

Comment:
Was removing all of this intentional?

Reply:
I'll double check that, but I think that came from a circular import, and the fix was suggested by Tim. However, it's worth me checking.